src/core/frame.c

```diff
@@ -1943,6 +1943,11 @@ QuicFrameLog(
         break;
     }
 
+    case QUIC_FRAME_RELIABLE_RESET_STREAM: {
+        // TODO: Add log.
```

Comment:
Leaving a note here so you remember to do this.

Reply:
Updated.